person.rb

```diff
@@ -0,0 +1,22 @@
+# Class Person file.
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    true if of_age? || @parent_permission
```

Comment:
- [ ] The logic of your method here is not complete because you are not handling the `falsy` case and therefore it will return `nil` instead of `false` if it is a falsy case. You better have to use directly in your return statement `of_age?` which is a [predicate method](http://ruby-for-beginners.rubymonstas.org/objects/predicates.html) (it returns a boolean, either true or false).

You would have something like
```
def can_use_services?
    of_age? || @parent_permission
end
```